### ISSUE TITLE
Docstring refers to parameters that have disapeared

### ISSUE THREAD

**Issue by sannant** [ContactEngineering/SurfaceTopography#180]
https://github.com/ContactEngineering/SurfaceTopography/blob/d68a8e8a765d9437d95748e815156d5ece0cd05b/SurfaceTopography/Uniform/PowerSpectrum.py#L66

I guess `nbins` became `nb_points` but what is `rmax` ?

**Comment by pastewka** [comment]
Yes, the docstring is wrong. This is passed on to `resample`, which has an argument `max_value`.

https://github.com/ContactEngineering/SurfaceTopography/blob/d68a8e8a765d9437d95748e815156d5ece0cd05b/SurfaceTopography/Support/Regression.py#L254
